platform/ios/src/MGLSDKUpdateChecker.mm

```diff
@@ -7,8 +7,7 @@ @implementation MGLSDKUpdateChecker
 + (void)checkForUpdates {
 #if TARGET_IPHONE_SIMULATOR
     // Abort if running in a playground.
-    if ([[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."] ||
-        NSProcessInfo.processInfo.mgl_isInterfaceBuilderDesignablesAgent) {
+    if ([[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."]) {
```

Comment:
Do we need to check for a playground in any of the places where we were previously checking for IB?

Reply:
Technically it’s not even necessary to check for playgrounds here — it’s just to stop a bit of console spam.